src/ratbag_emu/protocol/steelseries.py

```diff
@@ -102,7 +101,7 @@ def change_dpi(self, command, data, args):
 
         assert dpi_id == 1 or dpi_id == 2, 'Invalid DPI ID'
 
-        self.hw_profile.dpi[dpi_id - 1] = self.hw_profile.step * (dpi_steps + 1)
+        self.hw_profile.dpi[dpi_id - 1] = self.hw_profile.dpi_step * (dpi_steps + 1)
```

Comment:
Can you convert `step` into `dpi_step` in a separate commit, so we do not pollute this commit with millions of different things?